playbooks/vars/all.yml

```diff
@@ -27,3 +27,7 @@ default_gating_overrides:
   # too low for the liberty->mitaka upgrade job where more space is used by
   # additional packages, venvs, logs, etc.
   percent_used_critical_threshold: 95
+  # These neutron vars are required to allow instances to be pinged after an
+  # upgrade when using vxlan
+  neutron_l2_population: False
```

Comment:
The rpc-o default in newton for `neutron_l2_population` is `True`, are we concerned at all that we're not testing that default in any capacity?

Also, does this need to be set for kilo before the upgrade happens?  If so, I believe that config now all lives in repo at https://github.com/rcbops/rpc-openstack/tree/kilo/scripts/config.

Reply:
@mattt416 this is based on the testing @hughsaunders did to get existing floating IP addresses to continue to work after an upgrade. My understanding is no customer is using vxlan and so this configuration change is required specifically to test this part of the upgrade. An alternative option would be to have a script in osa-ops specifically for restarting the appropriate networking components after the upgrade when using vxlan with l2 pop.

Regarding where the configuration lives, I was not aware of this change until yesterday when I picked up RE-97, mnaio's do not currently use the new location. Depending on your first point, I will also set it in the patches to RPCO.

Reply:
@git-harry since we only do the persistent resource checks on an mnaio, how about moving this `neutron_l2_population` override to `playbooks/vars/onmetal.yml`?